src/profiler.h

```diff
@@ -134,6 +134,8 @@ class Profiler {
     void stopTimer();
     void timerLoop(void* timer_id);
 
+    void logSampleCollectionStatus(Arguments& args, u64 printed_samples_count, Writer& out);
```

Comment:
The name of the function is long but vague. I'd call it something like `logEmptyOutput` which is closer to its original purpose.